lib/log/appender/banner.js

```diff
@@ -2,12 +2,17 @@ var insertCSS = require('../../style/insertCss.js');
 var outDiv = null;
 var startTime = new Date().getTime();
 var OUTPUT_ID = 'logoutput';
+var overideTimeout;
+
+function timeoutWrap(fn, ms){
```

Comment:
Missed that one ye, will fix.
We should maybe remove the reset stuff as well then in the logger?